charts/testkube/values.yaml

```diff
@@ -355,6 +355,24 @@ testkube-api:
     digest: ""
     # -- Testkube API k8s secret for private registries
     pullSecrets: []
+
+  # Test Workflows image parameters
+  imageWorkflows:
+    # -- Test Workflows image registry
+    registry: docker.io
+    # -- Test Workflows image name
+    repositoryToolkit: kubeshop/testkube-tw-toolkit
+    # -- Test Workflows init image name
+    repositoryInit: kubeshop/testkube-tw-init
+    # Overrides the image tag whose default is the chart appVersion.
+    # tag: "latest"
+    # -- Test Workflows image tag
+    pullPolicy: IfNotPresent
```

Comment:
`pullPolicy`/`pullSecrets` are rather not required for them, as they are not used

Reply:
Do we have a default pull policy for the init & toolkit images?

If somebody must use private registries for all images, and re-push all of our public images, it would be nice to be able to define global pull secrets which would also by default get added to test workflows.

Reply:
For both questions: we don't

Reply:
Done